src/Services/Validators/Structure.php

```diff
@@ -43,10 +51,18 @@ public function summary(): array
         ];
     }
 
+    private function reader(): CSV|XLSX
+    {
+        return match ($this->extension) {
+            'csv' => new CSV($this->path),
+            default => new XLSX($this->path),
```

Comment:
'xlsx' => new XLSX($this->path),
default => throw new Enso Exception with message Unsupported import type